src/Stratis.Bitcoin/Base/ChainHeadersBehavior.cs

```diff
@@ -37,7 +37,7 @@ public class ChainHeadersBehavior : NetworkPeerBehavior
         public bool AutoSync { get; set; }
 
         /// <summary>
-        /// Information about the peer's announcement of its tip using "headers" message.
+        /// Our view of the peer's headers tip.
```

Comment:
you should preserve the information that we get this value from "headers" message

Reply:
Fixed.